### PR TITLE
Update Mavenir OWNERS for provider delivery

### PR DESCRIPTION
**Pull request by mmulholla**
Add providerDelivery: True

### CODE REVIEW

File: charts/partners/mavenir/amf/OWNERS [hunk 1]
```diff
@@ -2,6 +2,7 @@ chart:
   name: amf
   shortDescription: unknown
 publicPgpKey: unknown
+providerDelivery: True
 users: []
```
Comment (dperaza4dustbit) [inline charts/partners/mavenir/amf/OWNERS:6]: There is no users for this, I would wait for this to be fixed before we flag it provider delivery

**Review by dperaza4dustbit — CHANGES_REQUESTED** [review]
I would hold on this, see comment

File: charts/partners/mavenir/amf/OWNERS (hunk 1 above)
Comment (wying3) [reply charts/partners/mavenir/amf/OWNERS:6]: user has been added, please proceed the merge

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

**Pull request merged**